test/controllers/HomeControllerSpec.scala

```diff
@@ -339,5 +328,108 @@ class HomeControllerSpec extends BaseSpec with CurrentTaxYear {
 
       verify(controller.messageFrontendService, times(1)).getUnreadMessageCount(any())
     }
+
+  }
+
+  "Calling serviceCallResponses" should {
+
+    "return TaxComponentsDisabled status where there is not a PertaxContext and Nino" in new LocalSetup {
+      override def allowLowConfidenceSA: Boolean = false
+      val result = await(controller.serviceCallResponses(None, year))
+
+      result shouldBe ((TaxComponentsDisabledState, None, None))
+    }
+
+    "return TaxComponentsDisabled where taxComponents is not enabled" in new LocalSetup {
+      override def allowLowConfidenceSA: Boolean = false
+
+      when(controller.configDecorator.taxComponentsEnabled) thenReturn false
+
+      val userNino = Some(fakeNino)
```

Comment:
This in each test, you can pull it up to below this block

> "Calling serviceCallResponses" should {

And then just set it to None in the first test where there is no Nino